src/main/frontend/db_schema.cljs

```diff
@@ -60,12 +60,6 @@
    ;; "A", "B", "C"
    :block/priority {}
 
-   ;; TODO: remove
-   ;; 1, 2, 3, etc.
-   :block/level {}
```

Comment:
Just removed it from `retract-attributes` as `:block/level` shouldn't be transacted to the database. 
`:block/level` is used when calculating the `:block/parent` and `:block/left` from mldoc parsed AST.
It's also used when rendering blocks trees because some plugins and themes find it useful.